src/screens/Assistant/Assistant.tsx

```diff
@@ -436,6 +436,7 @@ const Assistant: React.FC<Props> = ({
       renderHeader={renderHeader}
       highlightComponent={newsBanner}
       onRefresh={refresh}
+      isRefreshing={searchState === 'searching'}
```

Comment:
Jeg tenkte egentlig `isRefreshing={searchState === 'searching' && !tripPatterns.length}`. Er det noen nedsider ved det?

Reply:
Syns å huske jeg hadde noe problemer med det, men fungerer bra her nå! Det løser også problemet på android når man laster flere, så da gjør vi det slik som du foreslår 😊

```suggestion
      isRefreshing={searchState === 'searching' && !tripPatterns.length}
```